draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -498,6 +516,16 @@ valid JWT Issuer Metadata configuration document before processing it.
 
 Additional considerations can be found in [@OWASP_SSRF].
 
+## Ecosystem-specific Public Key Verification Methods
+
+When defining ecosystem-specific rules for the verification of the public key,
+as outlined in (#public-key-for-issuer-signed-jwts-verification), it is critical
+that those rules maintain the integrity of the relationship between the `iss` value
+within the Issuer-signed JWT and the public keys of the Issuer.
+
+In situations where an attacker tampers with an SD-JWT VC, it is paramount
+to ensure that they cannot exert influence over the verification process while
+employing the same `iss` value.
```

Comment:
```suggestion
It MUST be ensured that for any given `iss` value, an attacker cannot influence what
kind of verification process is used. Otherwise, an attacker could attempt to
have the verifier use a verification process that was not intended by the Issuer
and where the attacker may have the opportunity to influcence the verification result
to his advantage.
```

Reply:
How about...

```
It MUST be ensured that for any given `iss` value, an attacker cannot influence the type of verification process used. Otherwise, an attacker could attempt to make the Verifier use a verification process not intended by the Issuer, allowing the attacker to potentially manipulate the verification result to their advantage.
```

Reply:
I used my suggested language in the updated PR.